### ISSUE TITLE
Android: application startup problems when the phone is in sleep mode (Display off)

### ISSUE THREAD

**Issue by Informate** [tidev/titanium-sdk#14097]
### I have searched and made sure there are no existing issues for the issue I am filing

- [X] I have searched the existing issues

### Description

When the application is started while the device is not active, specifically the screen has turned off and the phone has locked, some features of the application will not work properly.
This particular case happens, for example, when the application is launched through the command:

      ti build -p android --target device --device-id all

and the phone connected through the USB cable has entered idle mode due to inactivity.

In detail:
- Unbundled services do not start (especially on Android API 34).
- Some webviews using chart.js may return an error in the JS.
- Probably due to other errors in the code, the main app window sometimes does not open, and the apllication gets stuck on the splash screen.

I am unable to provide more specific information at this time.

### Expected Behavior

Works normally.

### Actual behavior

- Unbundled services do not start (especially on Android API 34).
- Some webviews using chart.js may return an error in the JS.
- Probably due to other errors in the code, the main app window sometimes does not open, and the apllication gets stuck on the splash screen.

### Reproducible sample

X

### Steps to reproduce

X

### Platform

Android

### SDK version you are using

12.4.0.GA

**Comment by m1ga** [comment]
I guess that is not a high priority issue as it just happens when you launch the app with adb/build commands and the phone is in sleep mode. That is nothing that would happen with the actual app later on, right. 

I quickly tested it with one bigger app I have here and I didn't see any issues starting a build while the phone is in sleep mode. I can see the logs and it even made API requests. When I wake up the phone and unlock it it will show the app and starts downloading images. I'm sure some stuff won't work that relies on focus or other events that might not fire when the phone is locked. But I wouldn't know how to debug that without waking up the phone :)

**Comment by Informate** [comment]
Yes, very very low priority! Just signaling the issues to let other know that this could happend!
